configure/jwt-authentication.md

```diff
@@ -1,23 +1,29 @@
 ---
-layout: draft
+layout: default
 title: JWT Authentication
 description_short: JWT Authentication
 description: JWT is a common authorization solution for web services.
 ---
 
-Some cloud services for TinyMCE require you to setup JWT authentication. This allows us to verify that you and your end user are allowed to access a particular feature. JWT is a common authorization solution for web services and is documented in more detail at the https://jwt.io/ website. This guide aims to show how to setup JWT authentication for the cloud services provided for TinyMCE.
+### Audience
+
+This section is intended to be used by developers with prior knowledge of JSON Web Tokens (or JWTs) in detail, including how they can be used for User Authentication and Session Management in a Web Application. There will be some coding involved on both the client-side and the server-side to configure JWT as per the instructions in this section.
```

Comment:
s/(or JWTs)/(JWT)/

Reply:
User Authentication, Session Management, and Web Application should not be capitalized in this case.